react/features/base/tracks/functions.js

```diff
@@ -58,15 +58,20 @@ export function createLocalTracksF(
         firefox_fake_device, // eslint-disable-line camelcase
         resolution
     } = state['features/base/config'];
-    const loadEffectsPromise = state['features/blur'].blurEnabled
-        ? getBlurEffect()
+    const loadEffectsPromise = !options.presenterMode
+        && state['features/blur'].blurEnabled ? getBlurEffect()
```

Comment:
This is bound to become more complex as more effects are added. How about adding the ability to disable or ignore effects, when creating a track?

Reply:
Done.